build.gradle

```diff
@@ -89,3 +93,7 @@ testClusters.integTest {
 run {
     useCluster testClusters.integTest
 }
+
+tasks.withType(RestIntegTestTask)*.configure {
+  classpath += files(project.configurations.runtimeClasspath.findAll { it.name.contains("log4j-core") })
```

Comment:
Just to explain what I found out:
 - the RestIntegTestTask needs log4j-core (it is used by test scaffolding)
 - in OpenSearch core, log4j-core  comes as part of `server` module but this dependency is `optional`
 - it seems like when dependencies are declared within the same multimodule build using `project`, the optional dependencies are included, otherwise - they are not

Reply:
PS: For some reasons, just adding the `log4j-core` to dependencies section does not solve the problem, may be because RestIntegTestPlugin includes only org.opensearch:opensearch module dependencies? I did not fully understand that yet ...

Reply:
Great find. This _is_ tricky

Reply:
Yeah, it took me a while to dig through classloaders & co to find out that :-(

Reply:
Daaam.... @reta Can you please open a bug in OpenSearch describing this mess and maybe suggesting how we can get out of it?

Reply:
👍 will do that